[100%] Testing the TCP connection...
Test project /mnt/i/DIY/CS144/sponge/build
        Start   1: t_wrapping_ints_cmp
  1/159 Test   #1: t_wrapping_ints_cmp ..............   Passed    0.01 sec
        Start   2: t_wrapping_ints_unwrap
  2/159 Test   #2: t_wrapping_ints_unwrap ...........   Passed    0.01 sec
        Start   3: t_wrapping_ints_wrap
  3/159 Test   #3: t_wrapping_ints_wrap .............   Passed    0.01 sec
        Start   4: t_recv_connect
  4/159 Test   #4: t_recv_connect ...................   Passed    0.05 sec
        Start   5: t_recv_transmit
  5/159 Test   #5: t_recv_transmit ..................   Passed    0.37 sec
        Start   6: t_recv_window
  6/159 Test   #6: t_recv_window ....................   Passed    0.03 sec
        Start   7: t_recv_reorder
  7/159 Test   #7: t_recv_reorder ...................   Passed    0.04 sec
        Start   8: t_recv_close
  8/159 Test   #8: t_recv_close .....................   Passed    0.04 sec
        Start   9: t_send_connect
  9/159 Test   #9: t_send_connect ...................   Passed    0.04 sec
        Start  10: t_send_transmit
 10/159 Test  #10: t_send_transmit ..................   Passed    0.34 sec
        Start  11: t_send_retx
 11/159 Test  #11: t_send_retx ......................   Passed    0.05 sec
        Start  12: t_send_window
 12/159 Test  #12: t_send_window ....................   Passed    0.19 sec
        Start  13: t_send_ack
 13/159 Test  #13: t_send_ack .......................   Passed    0.03 sec
        Start  14: t_send_close
 14/159 Test  #14: t_send_close .....................   Passed    0.03 sec
        Start  15: t_strm_reassem_single
 15/159 Test  #15: t_strm_reassem_single ............   Passed    0.02 sec
        Start  16: t_strm_reassem_seq
 16/159 Test  #16: t_strm_reassem_seq ...............   Passed    0.03 sec
        Start  17: t_strm_reassem_dup
 17/159 Test  #17: t_strm_reassem_dup ...............   Passed    0.03 sec
        Start  18: t_strm_reassem_holes
 18/159 Test  #18: t_strm_reassem_holes .............   Passed    0.02 sec
        Start  19: t_strm_reassem_many
 19/159 Test  #19: t_strm_reassem_many ..............   Passed    6.63 sec
        Start  20: t_strm_reassem_overlapping
 20/159 Test  #20: t_strm_reassem_overlapping .......   Passed    0.03 sec
        Start  21: t_strm_reassem_win
 21/159 Test  #21: t_strm_reassem_win ...............   Passed    7.26 sec
        Start  22: t_strm_reassem_cap
 22/159 Test  #22: t_strm_reassem_cap ...............   Passed    0.53 sec
        Start  23: t_byte_stream_construction
 23/159 Test  #23: t_byte_stream_construction .......   Passed    0.02 sec
        Start  24: t_byte_stream_one_write
 24/159 Test  #24: t_byte_stream_one_write ..........   Passed    0.02 sec
        Start  25: t_byte_stream_two_writes
 25/159 Test  #25: t_byte_stream_two_writes .........   Passed    0.02 sec
        Start  26: t_byte_stream_capacity
 26/159 Test  #26: t_byte_stream_capacity ...........   Passed    4.33 sec
        Start  27: t_byte_stream_many_writes
 27/159 Test  #27: t_byte_stream_many_writes ........   Passed    0.04 sec
        Start  28: t_webget
 28/159 Test  #28: t_webget .........................   Passed    1.65 sec
        Start  31: t_tcp_parser
 29/159 Test  #31: t_tcp_parser .....................   Passed    0.08 sec
        Start  32: t_ipv4_parser
 30/159 Test  #32: t_ipv4_parser ....................   Passed    0.09 sec
        Start  33: t_active_close
 31/159 Test  #33: t_active_close ...................***Failed    0.03 sec
TCP source port: 0
TCP dest port: 0
TCP seqno: 2
TCP ackno: 2
TCP doff: 5
Flags: urg: false ack: true psh: false rst: false syn: false fin: false
TCP winsize: fa00
TCP cksum: 0
TCP uptr: 0

TCP source port: 0
TCP dest port: 0
TCP seqno: 1
TCP ackno: 2
TCP doff: 5
Flags: urg: false ack: true psh: false rst: false syn: false fin: true
TCP winsize: fa00
TCP cksum: 0
TCP uptr: 0

TCP source port: 0
TCP dest port: 0
TCP seqno: 2
TCP ackno: 3
TCP doff: 5
Flags: urg: false ack: true psh: false rst: false syn: false fin: false
TCP winsize: fa00
TCP cksum: 0
TCP uptr: 0

TCP source port: 0
TCP dest port: 0
TCP seqno: 1
TCP ackno: 1
TCP doff: 5
Flags: urg: false ack: true psh: false rst: false syn: false fin: true
TCP winsize: fa00
TCP cksum: 0
TCP uptr: 0

TCP source port: 0
TCP dest port: 0
TCP seqno: 2
TCP ackno: 2
TCP doff: 5
Flags: urg: false ack: true psh: false rst: false syn: false fin: false
TCP winsize: fa00
TCP cksum: 0
TCP uptr: 0

Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,ackno=2,)

Failure message:
	The TCP an extra segment when it should not have

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      close
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=1,ackno=1,seqno=1,)
	Action:      packet arrives: Header(flags=A,seqno=1,ack=2,win=137) with no payload
	Expectation: TCP in state sender=`stream finished and fully acknowledged`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1
	Action:      5ms pass
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=2,win=137) with no payload
	Expectation: TCP in state sender=`stream finished and fully acknowledged`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1
	Expectation: Most recent incoming segment was 0 ms ago
	Action:      1ms pass
	Expectation: Most recent incoming segment was 1 ms ago
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)
	Action:      9990ms pass
	Expectation: Most recent incoming segment was 9991 ms ago
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=2,win=137) with no payload
	Expectation: Most recent incoming segment was 0 ms ago
	Action:      1ms pass

Note:
	test 5 failed: no ACK for 2nd FIN

Warning: Unclean shutdown of TCPConnection
The TCP an extra segment when it should not have

        Start  34: t_passive_close
 32/159 Test  #34: t_passive_close ..................***Failed    0.04 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=1,win=137) with no payload

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  36: t_ack_rst
 33/159 Test  #36: t_ack_rst ........................***Failed    0.09 sec
Test 1
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  38: t_ack_rst_win
 34/159 Test  #38: t_ack_rst_win ....................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  40: t_connect
 35/159 Test  #40: t_connect ........................   Passed    0.11 sec
        Start  42: t_listen
 36/159 Test  #42: t_listen .........................   Passed    0.05 sec
        Start  43: t_winsize
 37/159 Test  #43: t_winsize ........................***Failed    0.05 sec
Test Failure on expectation:
	Action:      write (57682 bytes with 57682 accepted) [6e, bd, bf, a4, b1, c6, 99, a7, f0, a7, f5, 33, 72, 91, 91, 6c, ...]

Failure message:
	57682 bytes should have been written but 0 were

List of steps that executed successfully:
	Action:      listen
	Action:      packet arrives: Header(flags=S,seqno=1121001610,ack=0,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,ackno=1121001611,win=29095,)
	Action:      packet arrives: Header(flags=A,seqno=1121001611,ack=2038102069,win=28841) with no payload
	Expectation: no (more) segments sent
	Expectation: TCP in state sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1

Warning: Unclean shutdown of TCPConnection
57682 bytes should have been written but 0 were

        Start  45: t_retx
 38/159 Test  #45: t_retx ...........................***Failed    0.06 sec
  check_segment
Test Failure on expectation:
	Expectation: segment sent with (A=1,payload_size=4,data="asdf",)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=245389413,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=245389413,ack=245389414,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=245389414,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not (in check_segment called from line 36)

        Start  46: t_retx_win
 39/159 Test  #46: t_retx_win .......................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2569511002,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2569511002,ack=2569511003,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2569511003,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass
	Action:      20ms pass
	Action:      write (4 bytes) [71, 77, 65, 72]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()s

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  47: t_loopback
 40/159 Test  #47: t_loopback .......................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: TCP has 1857 bytes in flight

Failure message:
	The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `1857`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2635284111,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2635284111,ack=2635284112,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2635284112,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2635284112,ack=2635284112,win=65000) with no payload
	Action:      write (1857 bytes) [88, b4, 4c, 19, 4c, a3, e7, 1e, 65, 5f, 24, e9, 5, 13, 89, a8, ...]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `1857`

        Start  48: t_loopback_win
 41/159 Test  #48: t_loopback_win ...................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2532095562,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2532095562,ack=2532095563,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2532095563,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2532095563,ack=2532095563,win=65000) with no payload
	Action:      write (5075 bytes) [4d, 86, 4a, f6, 5a, 5a, cd, e5, c, d1, b9, 13, cc, 11, 11, b4, ...]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  49: t_reorder
 42/159 Test  #49: t_reorder ........................***Failed    0.05 sec
Test Failure on expectation:
	segment sent with (A=1,)
Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=1308800468,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=4163374717,ack=1308800469,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=4163374718,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=4163402110,ack=1308800469,win=137) with 448 data bytes "<29><129>U<204><184>7<247><132>D<190>T<154><21>%]g..."

Note:
	test 1 failed: no ACK for datagram

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  50: t_address_dt
 43/159 Test  #50: t_address_dt .....................   Passed    0.03 sec
        Start  51: t_parser_dt
 44/159 Test  #51: t_parser_dt ......................   Passed    0.02 sec
        Start  52: t_socket_dt
 45/159 Test  #52: t_socket_dt ......................   Passed    0.03 sec
        Start  53: t_udp_client_send
 46/159 Test  #53: t_udp_client_send ................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21094... done.
new connection from 169.254.144.1:51166.
DEBUG: Outbound stream to 169.254.144.1:51166 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16307 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  54: t_udp_server_send
 47/159 Test  #54: t_udp_server_send ................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:2108... done.
new connection from 169.254.144.1:47909.
DEBUG: Outbound stream to 169.254.144.1:2108 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:47909 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16323 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 6cfd06a35f905b4e3fbf374916ffd69285fbcb4942a00055373285374cba46ef neq eeeedd24209d787175dbd53eec5132f91f4d1ae92363bd04a21e9e7a08beeb19 or 

        Start  55: t_udp_client_recv
 48/159 Test  #55: t_udp_client_recv ................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11421... done.
new connection from 169.254.144.1:53548.
DEBUG: Outbound stream to 169.254.144.1:11421 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:53548 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16344 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 608b7dd892afac56223bf1cfe495b9081913fbd027757317bbcfe87eaee61c52 neq 343e062ef50a0b685fc014363b343e3479f6ef9e597d40e1cf817ac9cadd05f8 or 

        Start  56: t_udp_server_recv
 49/159 Test  #56: t_udp_server_recv ................***Failed    0.85 sec
DEBUG: Connecting to 169.254.144.1:29130... DEBUG: Listening for incoming connection... done.
new connection from 169.254.144.1:41502.
DEBUG: Outbound stream to 169.254.144.1:41502 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16372 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  57: t_udp_client_dupl
 50/159 Test  #57: t_udp_client_dupl ................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9306... done.
new connection from 169.254.144.1:37492.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16389 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16391 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  58: t_udp_server_dupl
 51/159 Test  #58: t_udp_server_dupl ................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3756... done.
new connection from 169.254.144.1:33588.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16410 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16412 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  59: t_ucS_1M_32k
 52/159 Test  #59: t_ucS_1M_32k .....................***Failed    0.33 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:21094... done.
new connection from 169.254.144.1:59165.
DEBUG: Outbound stream to 169.254.144.1:59165 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:21094 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16434 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  60: t_ucS_128K_8K
 53/159 Test  #60: t_ucS_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23154... done.
new connection from 169.254.144.1:51787.
DEBUG: Outbound stream to 169.254.144.1:51787 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:23154 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16454 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  61: t_ucS_16_1
 54/159 Test  #61: t_ucS_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:12251... done.
new connection from 169.254.144.1:40038.
DEBUG: Outbound stream to 169.254.144.1:40038 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16474 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  62: t_ucS_32K_d
 55/159 Test  #62: t_ucS_32K_d ......................***Failed    0.32 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:24758... done.
new connection from 169.254.144.1:32788.
DEBUG: Outbound stream to 169.254.144.1:32788 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:24758 finished cleanly.
DEBUG: Waiting for clean shutdown... DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16492 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  63: t_ucR_1M_32k
 56/159 Test  #63: t_ucR_1M_32k .....................***Failed    0.63 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:30870... done.
new connection from 169.254.144.1:52656.
DEBUG: Outbound stream to 169.254.144.1:30870 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:52656 finished cleanly.
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 228: 16513 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: d3ed5f717b57b68a7f48da710e9646828b68a4604f58d0599aa72a685b88944d neq 087cfedc78aa07498934dcd841384aec86e0ddf5da747aa4d469a65daf744820 or 

        Start  64: t_ucR_128K_8K
 57/159 Test  #64: t_ucR_128K_8K ....................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14161... done.
new connection from 169.254.144.1:52686.
DEBUG: Outbound stream to 169.254.144.1:14161 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:52686 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16536 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 52dfba37bd914bcdc74c6fc62573fa026c61912d96a32d9943962ddedfb5c6c6 neq 85a06c0afbd620e211bf4c66a24c87ddf6a00899506f34b80f9ccd1dcd5510d2 or 

        Start  65: t_ucR_16_1
 58/159 Test  #65: t_ucR_16_1 .......................***Failed    0.32 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29687... done.
new connection from 169.254.144.1:49105.
DEBUG: Outbound stream to 169.254.144.1:29687 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:49105 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16561 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: cee9c480d065d6b2f843c952c3e1f3e96558ecf0050ba6724aadeb0b81582093 neq 9423e03f82090856f3fc0fdb1dad7e2fb3aeba5dcf4da9c7fa2be45af3f88d6b or 

        Start  66: t_ucR_32K_d
 59/159 Test  #66: t_ucR_32K_d ......................***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22755... done.
new connection from 169.254.144.1:36050.
DEBUG: Outbound stream to 169.254.144.1:22755 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:36050 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16582 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 578b65f2e3c5ca1f434c6bc0db26fd25e5b3207dd3714725729a9b02bc2f4d46 neq e04b290ba442ac7d4d0c5da31c7f93633f70a212d8a665858cb0b34a472ba68a or 

        Start  67: t_ucD_1M_32k
 60/159 Test  #67: t_ucD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11200... done.
new connection from 169.254.144.1:44303.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16608 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  68: t_ucD_128K_8K
 61/159 Test  #68: t_ucD_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:12839... done.
new connection from 169.254.144.1:52522.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16632 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  69: t_ucD_16_1
 62/159 Test  #69: t_ucD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19256... done.
new connection from 169.254.144.1:53161.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16660 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16662 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  70: t_ucD_32K_d
 63/159 Test  #70: t_ucD_32K_d ......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:24635... done.
new connection from 169.254.144.1:60347.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16679 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16681 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  71: t_usS_1M_32k
 64/159 Test  #71: t_usS_1M_32k .....................***Failed    0.36 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7073... done.
new connection from 169.254.144.1:56437.
DEBUG: Outbound stream to 169.254.144.1:7073 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:56437 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16699 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 07a49bd937c9d5bbe936a86fa2565209f117ce36358772a155abb7bbed42c636 neq f76f84c8cee6268b48ac6587b4939766e6589cac7032e4cb42289f6d358cd16e or 

        Start  72: t_usS_128K_8K
 65/159 Test  #72: t_usS_128K_8K ....................***Failed    0.39 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9482... done.
new connection from 169.254.144.1:41651.
DEBUG: Outbound stream to 169.254.144.1:9482 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:41651 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16722 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 936e26a323e36668b9a890f0b7dcde627348db6a3b169e369cfbc432031e7f24 neq d9435a82997af5d049c121ffc6aba57ee02c6b84936d4de4497fbabea1042308 or 

        Start  73: t_usS_16_1
 66/159 Test  #73: t_usS_16_1 .......................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:15062... done.
new connection from 169.254.144.1:56739.
DEBUG: Outbound stream to 169.254.144.1:15062 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:56739 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16745 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 68b754d2a61b1d0eee3585d803d9d92b78f6fd5946798cdea5d553a4d90c7eca neq 4eb97d27bb87978505704160905ce19be16217e64e8f5165f0dd8466aac86b7b or 

        Start  74: t_usS_32K_d
 67/159 Test  #74: t_usS_32K_d ......................***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5165... done.
new connection from 169.254.144.1:46987.
DEBUG: Outbound stream to 169.254.144.1:5165 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:46987 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16766 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: d2e878228a3e0eb95e0c1224c686bad4dafc177a370b19a165bf1ea68bbf537b neq 95d7e818cdcc3cb7a230fbfa6c1eef2cf122184bb8f3102023849b4a20223610 or 

        Start  75: t_usR_1M_32k
 68/159 Test  #75: t_usR_1M_32k .....................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22160... done.
new connection from 169.254.144.1:49896.
DEBUG: Outbound stream to 169.254.144.1:49896 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:22160 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16794 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  76: t_usR_128K_8K
 69/159 Test  #76: t_usR_128K_8K ....................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29217... done.
new connection from 169.254.144.1:58412.
DEBUG: Outbound stream to 169.254.144.1:58412 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:29217 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16814 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  77: t_usR_16_1
 70/159 Test  #77: t_usR_16_1 .......................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:2991... done.
new connection from 169.254.144.1:45425.
DEBUG: Outbound stream to 169.254.144.1:45425 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16834 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  78: t_usR_32K_d
 71/159 Test  #78: t_usR_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29250... done.
new connection from 169.254.144.1:37016.
DEBUG: Outbound stream to 169.254.144.1:37016 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:29250 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16852 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  79: t_usD_1M_32k
 72/159 Test  #79: t_usD_1M_32k .....................***Failed    0.27 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:30665... done.
new connection from 169.254.144.1:34499.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16873 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  80: t_usD_128K_8K
 73/159 Test  #80: t_usD_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5026... done.
new connection from 169.254.144.1:35152.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16892 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16894 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  81: t_usD_16_1
 74/159 Test  #81: t_usD_16_1 .......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7109... done.
new connection from 169.254.144.1:45985.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16915 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  82: t_usD_32K_d
 75/159 Test  #82: t_usD_32K_d ......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32085... done.
new connection from 169.254.144.1:36811.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16937 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  83: t_ucS_128K_8K_l
 76/159 Test  #83: t_ucS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14150... 

        Start  84: t_ucS_128K_8K_L
 77/159 Test  #84: t_ucS_128K_8K_L ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22197... done.
new connection from 169.254.144.1:60354.
DEBUG: Outbound stream to 169.254.144.1:60354 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:22197 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 16989 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  85: t_ucS_128K_8K_lL
 78/159 Test  #85: t_ucS_128K_8K_lL .................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:27974... done.
new connection from 169.254.144.1:59073.
DEBUG: Outbound stream to 169.254.144.1:59073 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:27974 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17009 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  86: t_ucR_128K_8K_l
 79/159 Test  #86: t_ucR_128K_8K_l ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25225... 

        Start  87: t_ucR_128K_8K_L
 80/159 Test  #87: t_ucR_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:30266... 

        Start  88: t_ucR_128K_8K_lL
 81/159 Test  #88: t_ucR_128K_8K_lL .................***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:16470... done.
new connection from 169.254.144.1:49842.
DEBUG: Outbound stream to 169.254.144.1:16470 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:49842 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17097 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: df503f83dac9a60130c83db35ce145588d3f4debb81f31e87612f0edf0187cdc neq cd7bdba0edf91b19cd83540fd267c9139ccd56e1038d94051af0ce58581cde36 or 

        Start  89: t_ucD_128K_8K_l
 82/159 Test  #89: t_ucD_128K_8K_l ..................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32514... done.
new connection from 169.254.144.1:46332.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17126 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  90: t_ucD_128K_8K_L
 83/159 Test  #90: t_ucD_128K_8K_L ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:1416... done.
new connection from 169.254.144.1:59877.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17147 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 17145 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  91: t_ucD_128K_8K_lL
 84/159 Test  #91: t_ucD_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17781... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17168 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  92: t_usS_128K_8K_l
 85/159 Test  #92: t_usS_128K_8K_l ..................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25012... done.
new connection from 169.254.144.1:53522.
DEBUG: Outbound stream to 169.254.144.1:25012 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:53522 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17185 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 5e2f20233f24c6ba040b77f2d24ab0468a42283093c85cf16edb08bb18011766 neq 098d8bd41faa83c1b2a6194e8570fbecdb9728ef5ef5d9a6a0f194ec52e7ea6c or 

        Start  93: t_usS_128K_8K_L
 86/159 Test  #93: t_usS_128K_8K_L ..................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14364... done.
new connection from 169.254.144.1:46575.
DEBUG: Outbound stream to 169.254.144.1:14364 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:46575 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17208 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 9bea9225d963da388f6c548215b79f480edb5ac6290c2a0c5c570866aa6a936c neq cf3979cd8092b0e444de664c5760059cd903882e092ac1127fa84a65776cd6f3 or 

        Start  94: t_usS_128K_8K_lL
 87/159 Test  #94: t_usS_128K_8K_lL .................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9310... done.
DEBUG: Outbound stream to 169.254.144.1:9310 finished (1 byte still in flight).
new connection from 169.254.144.1:60601.
DEBUG: Inbound stream from 169.254.144.1:60601 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17231 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: ed0b51deb59abf8ecff99871c84fbe2eefb0d82358fc95d2b66685dac9f6bb9d neq b108ffde0f596193d916ceaf15e62893ec8dfdc33067ab410f4d49ab3b2ee08f or 

        Start  95: t_usR_128K_8K_l
 88/159 Test  #95: t_usR_128K_8K_l ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:17610... done.
new connection from 169.254.144.1:55695.
DEBUG: Outbound stream to 169.254.144.1:55695 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:17610 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17256 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  96: t_usR_128K_8K_L
 89/159 Test  #96: t_usR_128K_8K_L ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:1173... done.
new connection from 169.254.144.1:40290.
DEBUG: Outbound stream to 169.254.144.1:40290 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:1173 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17276 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  97: t_usR_128K_8K_lL
 90/159 Test  #97: t_usR_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9769... done.
new connection from 169.254.144.1:34374.
DEBUG: Outbound stream to 169.254.144.1:34374 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.1:9769 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17299 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  98: t_usD_128K_8K_l
 91/159 Test  #98: t_usD_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5254... done.
new connection from 169.254.144.1:42327.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17320 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  99: t_usD_128K_8K_L
 92/159 Test  #99: t_usD_128K_8K_L ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:13783... done.
new connection from 169.254.144.1:50339.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17341 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 100: t_usD_128K_8K_lL
 93/159 Test #100: t_usD_128K_8K_lL .................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:6882... done.
new connection from 169.254.144.1:51059.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17362 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 101: t_ipv4_client_send
 94/159 Test #101: t_ipv4_client_send ...............***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:17628... done.
new connection from 169.254.144.9:50192.
DEBUG: Outbound stream to 169.254.144.9:50192 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17380 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 102: t_ipv4_server_send
 95/159 Test #102: t_ipv4_server_send ...............***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:8618... done.
DEBUG: Outbound stream to 169.254.144.9:8618 finished (1 byte still in flight).
new connection from 169.254.145.9:34835.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17396 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: a3da7bce64f85477b342fd97f7a3a75f80c01b43030456de88693d921580e38c neq e827abd40b27d3dab3063e996ca259e1c12fe95a6cda0257f825de9082392f2c or 

        Start 103: t_ipv4_client_recv
 96/159 Test #103: t_ipv4_client_recv ...............***Failed    0.34 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:17757... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:49383.
169.254.145.9:17757 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17418 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: e1de8da5fb1b8b6b51e5349da32f56185a6b8d8c4ef0f438343d992265267161 neq 89abbb5d7021824c4118fed0ac8fad1ab508f9058b1287b682f9d81aa5fccfb2 or 

        Start 104: t_ipv4_server_recv
 97/159 Test #104: t_ipv4_server_recv ...............***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:27392... done.
new connection from 169.254.145.9:50252.
DEBUG: Outbound stream to 169.254.145.9:50252 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17444 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 105: t_ipv4_client_dupl
 98/159 Test #105: t_ipv4_client_dupl ...............***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:11752... done.
new connection from 169.254.144.9:12771.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17463 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 106: t_ipv4_server_dupl
 99/159 Test #106: t_ipv4_server_dupl ...............***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:25206... done.
new connection from 169.254.145.9:1222.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17482 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 107: t_icS_1M_32k
100/159 Test #107: t_icS_1M_32k .....................***Failed    0.28 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:3248... done.
new connection from 169.254.144.9:51382.
DEBUG: Outbound stream to 169.254.144.9:51382 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:3248 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17502 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 108: t_icS_128K_8K
101/159 Test #108: t_icS_128K_8K ....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:9758... done.
new connection from 169.254.144.9:22916.
DEBUG: Outbound stream to 169.254.144.9:22916 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:9758 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17522 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 109: t_icS_16_1
102/159 Test #109: t_icS_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:29448... done.
new connection from 169.254.144.9:50331.
DEBUG: Outbound stream to 169.254.144.9:50331 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17542 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 110: t_icS_32K_d
103/159 Test #110: t_icS_32K_d ......................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:6242... done.
new connection from 169.254.144.9:5976.
DEBUG: Outbound stream to 169.254.144.9:5976 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:6242 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17560 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 111: t_icR_1M_32k
104/159 Test #111: t_icR_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:28908... done.
DEBUG: Outbound stream to 169.254.145.9:28908 finished (1 byte still in flight).
new connection from 169.254.144.9:36572.
DEBUG: Inbound stream from 169.254.144.9:36572 finished cleanly.
DEBUG: Outbound stream to 169.254.145.9:28908 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 112: t_icR_128K_8K
105/159 Test #112: t_icR_128K_8K ....................***Failed    0.33 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:26721... done.
DEBUG: Outbound stream to 169.254.145.9:26721 finished (1 byte still in flight).
new connection from 169.254.144.9:25658.
DEBUG: Inbound stream from 169.254.144.9:25658 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17612 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: d16505e545cca84557ab6c64a26603d2d7b124ad7a44e4c8ab7052f4af84428c neq 71ca6d74767a036665fd5e5690a060d3f3db8733aac0469da10f988228c4c376 or 

        Start 113: t_icR_16_1
106/159 Test #113: t_icR_16_1 .......................***Failed    0.32 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22276... done.
DEBUG: Outbound stream to 169.254.145.9:22276new connection from 169.254.144.9:53821.
 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17635 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: d8510b82fad0b9677bb4f6fd18cd81d14a3f8052aa7ef9365f63510dffbc9ac7 neq 96bdec039b769842027a3b460504beeff28876011a9d35cb42e4c7df64c6393e or 

        Start 114: t_icR_32K_d
107/159 Test #114: t_icR_32K_d ......................***Failed    0.32 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:20355... done.
DEBUG: Outbound stream to new connection from 169.254.145.9:20355 finished (169.254.144.9:3935.
1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:3935 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17659 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 791565ada1cbe7167682794bb41785d9b702232fc56e46f03f2be6b85dc8dac9 neq 5d27684fb0572322e29ea37248fb1c484be11ec5348fbb7565060065795df441 or 

        Start 115: t_icD_1M_32k
108/159 Test #115: t_icD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:6549... done.
new connection from 169.254.144.9:5021.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17685 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 116: t_icD_128K_8K
109/159 Test #116: t_icD_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:14101... done.
new connection from 169.254.144.9:18356.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17706 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 117: t_icD_16_1
110/159 Test #117: t_icD_16_1 .......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23681... done.
new connection from 169.254.144.9:28664.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17725 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17727 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 118: t_icD_32K_d
111/159 Test #118: t_icD_32K_d ......................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22879... done.
new connection from 169.254.144.9:57490.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17744 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17746 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 119: t_isS_1M_32k
112/159 Test #119: t_isS_1M_32k .....................***Failed    0.31 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:22640... done.
DEBUG: Outbound stream to 169.254.144.9:22640 finished (1new connection from  byte still in flight).
169.254.145.9:11321.
DEBUG: Inbound stream from 169.254.145.9:11321 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17764 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: f2074bb73f12c501015a4b27879e1e9f1e6b6f4a639f1c90e769ee7af95921f3 neq 430890a4a5b0e1802759cff48e27b93823de634bfcc68c25b5f5b5f5bdd21a17 or 

        Start 120: t_isS_128K_8K
113/159 Test #120: t_isS_128K_8K ....................***Failed    0.31 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:29015... done.
new connection from 169.254.145.9:22866.
DEBUG: Outbound stream to 169.254.144.9:29015 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:22866 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17787 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 1937f63823896253ea7831ccef9fbacf0be98bdf565be03359165abf26918be6 neq a46f1c349985012780c887c637dc43ec10be71858c273d9acbe2723eaeeec598 or 

        Start 121: t_isS_16_1
114/159 Test #121: t_isS_16_1 .......................***Failed    0.33 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:5744... done.
DEBUG: Outbound stream to 169.254.144.9:5744 finished (1 byte still in flight).
new connection from 169.254.145.9:9821.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17813 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 0d5b2cc8d88e7ad4c9db72404051980800a774e06bceca547178515a04551f00 neq 4967f54d8d01d4d07f23fc0f708c2423a0fac645a1e2c42ecbf3596643919156 or 

        Start 122: t_isS_32K_d
115/159 Test #122: t_isS_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:7181... done.
DEBUG: Outbound stream to 169.254.144.9:7181 finished (1 byte still in flight).
new connection from 169.254.145.9:54850.
DEBUG: Inbound stream from 169.254.145.9:54850 finished cleanly.
DEBUG: Outbound stream to 169.254.144.9:7181 has been fully acknowledged.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 123: t_isR_1M_32k
116/159 Test #123: t_isR_1M_32k .....................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:18264... done.
new connection from 169.254.145.9:60613.
DEBUG: Outbound stream to 169.254.145.9:60613 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:18264 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17870 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 124: t_isR_128K_8K
117/159 Test #124: t_isR_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:9525... done.
new connection from 169.254.145.9:55128.
DEBUG: Outbound stream to 169.254.145.9:55128 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:9525 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17890 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 125: t_isR_16_1
118/159 Test #125: t_isR_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:33184... done.
new connection from 169.254.145.9:39921.
DEBUG: Outbound stream to 169.254.145.9:39921 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17910 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 126: t_isR_32K_d
119/159 Test #126: t_isR_32K_d ......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15781... done.
new connection from 169.254.145.9:46051.
DEBUG: Outbound stream to 169.254.145.9:46051 finished (1 byte still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17928 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 127: t_isD_1M_32k
120/159 Test #127: t_isD_1M_32k .....................***Failed    0.28 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:8499... done.
new connection from 169.254.145.9:34429.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17947 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17949 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 128: t_isD_128K_8K
121/159 Test #128: t_isD_128K_8K ....................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:32568... done.
new connection from 169.254.145.9:26652.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17968 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17970 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 129: t_isD_16_1
122/159 Test #129: t_isD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15427... done.
new connection from 169.254.145.9:2440.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17989 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 17991 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 130: t_isD_32K_d
123/159 Test #130: t_isD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:2216... done.
new connection from 169.254.145.9:55311.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18013 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156: 18008 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 131: t_icS_128K_8K_l
124/159 Test #131: t_icS_128K_8K_l ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:27276... done.
new connection from 169.254.144.9:45395.
DEBUG: Outbound stream to 169.254.144.9:45395 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:27276 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18033 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 132: t_icS_128K_8K_L
125/159 Test #132: t_icS_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:18180... done.
new connection from 169.254.144.9:54168.
DEBUG: Outbound stream to 169.254.144.9:54168 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:18180 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18053 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 133: t_icS_128K_8K_lL
126/159 Test #133: t_icS_128K_8K_lL .................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23920... done.
new connection from 169.254.144.9:29245.
DEBUG: Outbound stream to 169.254.144.9:29245 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:23920 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18073 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 134: t_icR_128K_8K_l
127/159 Test #134: t_icR_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:13042... done.
DEBUG: Outbound stream to 169.254.145.9:13042 finished (1 byte still in flight).
new connection from 169.254.144.9:27196.
DEBUG: Inbound stream from 169.254.144.9:27196 finished cleanly.
DEBUG: Outbound stream to 169.254.145.9:13042 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 135: t_icR_128K_8K_L
128/159 Test #135: t_icR_128K_8K_L ..................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:29601... done.
new connection from 169.254.144.9:44314.
DEBUG: Outbound stream to 169.254.145.9:29601 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:44314 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18132 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: be51c2795910fd56de2eab93ff149e045cb3dcf87694ca3458e2750bf78e71e3 neq bee0dbbbea21fad8f01388a57d5a430aded918866f783a38b50ddee45fffda23 or 

        Start 136: t_icR_128K_8K_lL
129/159 Test #136: t_icR_128K_8K_lL .................***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:6701... done.
DEBUG: Outbound stream to 169.254.145.9:6701 finished (1 byte still in flight).
new connection from 169.254.144.9:8382.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18155 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 7a59b38b2364d6fac1223d24755fca1c6e0fd8f0525e6aa92749182171f63477 neq ec3b4327a43efa6d82eb69b66fd4b8a81306b303e4a96340e11e10fbcdabd66d or 

        Start 137: t_icD_128K_8K_l
130/159 Test #137: t_icD_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:7675... done.
new connection from 169.254.144.9:55388.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18181 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 138: t_icD_128K_8K_L
131/159 Test #138: t_icD_128K_8K_L ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:21409... done.
new connection from 169.254.144.9:47852.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18200 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18202 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 139: t_icD_128K_8K_lL
132/159 Test #139: t_icD_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:23954... done.
new connection from 169.254.144.9:30486.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18223 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 140: t_isS_128K_8K_l
133/159 Test #140: t_isS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:9500... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:9500 finished (169.254.145.9:30872.
1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:30872 finished cleanly.
DEBUG: Outbound stream to 169.254.144.9:9500 has been fully acknowledged.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 141: t_isS_128K_8K_L
134/159 Test #141: t_isS_128K_8K_L ..................***Failed    0.32 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:5671... done.
DEBUG: Outbound stream to new connection from 169.254.145.9:61253.
169.254.144.9:5671 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.145.9:61253 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18275 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: 03ce2b7e38aed896105fd34eb1f660a1c8f548926eb356a5cb8456c61bd728ea neq 6d2ac10a6dc4dffc1176cb4db45e4d8b58bf3500e2b3473c11f95e64c3485539 or 

        Start 142: t_isS_128K_8K_lL
135/159 Test #142: t_isS_128K_8K_lL .................***Failed    0.30 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:2975... done.
DEBUG: Outbound stream to 169.254.144.9:2975 finished (1 byte still in flight).
new connection from 169.254.145.9:11653.
DEBUG: Inbound stream from 169.254.145.9:11653 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
DEBUG: TCP connection finished cleanly.
DEBUG: Waiting for clean shutdown... done.
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18298 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
ERROR: c80c10b76e0683b5929c1b4c7a3729fcd0eba6e320eb3c86b41e27e239d6f7ae neq b857819209147397ac3cd7d94f228623b8b468d5bbfc42c9c48e5d74ceaa9fc4 or 

        Start 143: t_isR_128K_8K_l
136/159 Test #143: t_isR_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15472... done.
new connection from 169.254.145.9:45097.
DEBUG: Outbound stream to 169.254.145.9:45097 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:15472 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18326 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 144: t_isR_128K_8K_L
137/159 Test #144: t_isR_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:11266... done.
new connection from 169.254.145.9:30822.
DEBUG: Outbound stream to 169.254.145.9:30822 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:11266 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18346 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 145: t_isR_128K_8K_lL
138/159 Test #145: t_isR_128K_8K_lL .................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:18583... done.
new connection from 169.254.145.9:28191.
DEBUG: Outbound stream to 169.254.145.9:28191 finished (1 byte still in flight).
DEBUG: Inbound stream from 169.254.144.9:18583 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18366 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 146: t_isD_128K_8K_l
139/159 Test #146: t_isD_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:15905... done.
new connection from 169.254.145.9:8509.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18387 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 147: t_isD_128K_8K_L
140/159 Test #147: t_isD_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:20241... 

        Start 148: t_isD_128K_8K_lL
141/159 Test #148: t_isD_128K_8K_lL .................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:29330... done.
new connection from 169.254.145.9:47596.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18439 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18441 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 149: t_icnS_128K_8K_l
142/159 Test #149: t_icnS_128K_8K_l .................***Failed    0.25 sec
DEBUG: Connecting to 169.254.144.1:1938... done.
DEBUG: Inbound stream from 169.254.144.1:1938 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18461 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 150: t_icnS_128K_8K_L
143/159 Test #150: t_icnS_128K_8K_L .................***Failed    0.23 sec
DEBUG: Connecting to 169.254.144.1:9977... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18480 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 151: t_icnS_128K_8K_lL
144/159 Test #151: t_icnS_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Connecting to 169.254.144.1:8554... 

        Start 152: t_icnR_128K_8K_l
145/159 Test #152: t_icnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:7625... done.
DEBUG: Outbound stream to 169.254.144.1:7625 finished (1 byte still in flight).

        Start 153: t_icnR_128K_8K_L
146/159 Test #153: t_icnR_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Connecting to 169.254.144.1:13532... done.
DEBUG: Outbound stream to 169.254.144.1:13532 finished (1 byte still in flight).
DEBUG: Outbound stream to 169.254.144.1:13532 has been fully acknowledged.

        Start 154: t_icnR_128K_8K_lL
147/159 Test #154: t_icnR_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:8301... done.
DEBUG: Outbound stream to 169.254.144.1:8301 finished (1 byte still in flight).
DEBUG: Outbound stream to 169.254.144.1:8301 has been fully acknowledged.

        Start 155: t_icnD_128K_8K_l
148/159 Test #155: t_icnD_128K_8K_l .................***Failed    0.28 sec
DEBUG: Connecting to 169.254.144.1:12435... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18684 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 156: t_icnD_128K_8K_L
149/159 Test #156: t_icnD_128K_8K_L .................***Failed    0.26 sec
DEBUG: Connecting to 169.254.144.1:28026... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18707 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 157: t_icnD_128K_8K_lL
150/159 Test #157: t_icnD_128K_8K_lL ................***Failed    0.26 sec
DEBUG: Connecting to 169.254.144.1:28059... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110: 18727 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 158: t_isnS_128K_8K_l
151/159 Test #158: t_isnS_128K_8K_l .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:52100.
DEBUG: Inbound stream from 169.254.144.1:52100 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 159: t_isnS_128K_8K_L
152/159 Test #159: t_isnS_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... 

        Start 160: t_isnS_128K_8K_lL
153/159 Test #160: t_isnS_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:46366.
DEBUG: Inbound stream from 169.254.144.1:46366 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 161: t_isnR_128K_8K_l
154/159 Test #161: t_isnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:36994.
DEBUG: Outbound stream to 169.254.144.1:36994 finished (1 byte still in flight).
DEBUG: Outbound stream to 169.254.144.1:36994 has been fully acknowledged.

        Start 162: t_isnR_128K_8K_L
155/159 Test #162: t_isnR_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:44934.
DEBUG: Outbound stream to 169.254.144.1:44934 finished (1 byte still in flight).
DEBUG: Outbound stream to 169.254.144.1:44934 has been fully acknowledged.

        Start 163: t_isnR_128K_8K_lL
156/159 Test #163: t_isnR_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:52266.
DEBUG: Outbound stream to 169.254.144.1:52266 finished (1 byte still in flight).
DEBUG: Outbound stream to 169.254.144.1:52266 has been fully acknowledged.

        Start 164: t_isnD_128K_8K_l
157/159 Test #164: t_isnD_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:57508.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 165: t_isnD_128K_8K_L
158/159 Test #165: t_isnD_128K_8K_L .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:56610.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 166: t_isnD_128K_8K_lL
159/159 Test #166: t_isnD_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... 


22% tests passed, 124 tests failed out of 159

Total Test time (real) = 263.71 sec

The following tests FAILED:
	 33 - t_active_close (Failed)
	 34 - t_passive_close (Failed)
	 36 - t_ack_rst (Failed)
	 38 - t_ack_rst_win (Failed)
	 43 - t_winsize (Failed)
	 45 - t_retx (Failed)
	 46 - t_retx_win (Failed)
	 47 - t_loopback (Failed)
	 48 - t_loopback_win (Failed)
	 49 - t_reorder (Failed)
	 53 - t_udp_client_send (Failed)
	 54 - t_udp_server_send (Failed)
	 55 - t_udp_client_recv (Failed)
	 56 - t_udp_server_recv (Failed)
	 57 - t_udp_client_dupl (Failed)
	 58 - t_udp_server_dupl (Failed)
	 59 - t_ucS_1M_32k (Failed)
	 60 - t_ucS_128K_8K (Failed)
	 61 - t_ucS_16_1 (Failed)
	 62 - t_ucS_32K_d (Failed)
	 63 - t_ucR_1M_32k (Failed)
	 64 - t_ucR_128K_8K (Failed)
	 65 - t_ucR_16_1 (Failed)
	 66 - t_ucR_32K_d (Failed)
	 67 - t_ucD_1M_32k (Failed)
	 68 - t_ucD_128K_8K (Failed)
	 69 - t_ucD_16_1 (Failed)
	 70 - t_ucD_32K_d (Failed)
	 71 - t_usS_1M_32k (Failed)
	 72 - t_usS_128K_8K (Failed)
	 73 - t_usS_16_1 (Failed)
	 74 - t_usS_32K_d (Failed)
	 75 - t_usR_1M_32k (Failed)
	 76 - t_usR_128K_8K (Failed)
	 77 - t_usR_16_1 (Failed)
	 78 - t_usR_32K_d (Failed)
	 79 - t_usD_1M_32k (Failed)
	 80 - t_usD_128K_8K (Failed)
	 81 - t_usD_16_1 (Failed)
	 82 - t_usD_32K_d (Failed)
	 83 - t_ucS_128K_8K_l (Timeout)
	 84 - t_ucS_128K_8K_L (Failed)
	 85 - t_ucS_128K_8K_lL (Failed)
	 86 - t_ucR_128K_8K_l (Timeout)
	 87 - t_ucR_128K_8K_L (Timeout)
	 88 - t_ucR_128K_8K_lL (Failed)
	 89 - t_ucD_128K_8K_l (Failed)
	 90 - t_ucD_128K_8K_L (Failed)
	 91 - t_ucD_128K_8K_lL (Failed)
	 92 - t_usS_128K_8K_l (Failed)
	 93 - t_usS_128K_8K_L (Failed)
	 94 - t_usS_128K_8K_lL (Failed)
	 95 - t_usR_128K_8K_l (Failed)
	 96 - t_usR_128K_8K_L (Failed)
	 97 - t_usR_128K_8K_lL (Failed)
	 98 - t_usD_128K_8K_l (Failed)
	 99 - t_usD_128K_8K_L (Failed)
	100 - t_usD_128K_8K_lL (Failed)
	101 - t_ipv4_client_send (Failed)
	102 - t_ipv4_server_send (Failed)
	103 - t_ipv4_client_recv (Failed)
	104 - t_ipv4_server_recv (Failed)
	105 - t_ipv4_client_dupl (Failed)
	106 - t_ipv4_server_dupl (Failed)
	107 - t_icS_1M_32k (Failed)
	108 - t_icS_128K_8K (Failed)
	109 - t_icS_16_1 (Failed)
	110 - t_icS_32K_d (Failed)
	111 - t_icR_1M_32k (Timeout)
	112 - t_icR_128K_8K (Failed)
	113 - t_icR_16_1 (Failed)
	114 - t_icR_32K_d (Failed)
	115 - t_icD_1M_32k (Failed)
	116 - t_icD_128K_8K (Failed)
	117 - t_icD_16_1 (Failed)
	118 - t_icD_32K_d (Failed)
	119 - t_isS_1M_32k (Failed)
	120 - t_isS_128K_8K (Failed)
	121 - t_isS_16_1 (Failed)
	122 - t_isS_32K_d (Timeout)
	123 - t_isR_1M_32k (Failed)
	124 - t_isR_128K_8K (Failed)
	125 - t_isR_16_1 (Failed)
	126 - t_isR_32K_d (Failed)
	127 - t_isD_1M_32k (Failed)
	128 - t_isD_128K_8K (Failed)
	129 - t_isD_16_1 (Failed)
	130 - t_isD_32K_d (Failed)
	131 - t_icS_128K_8K_l (Failed)
	132 - t_icS_128K_8K_L (Failed)
	133 - t_icS_128K_8K_lL (Failed)
	134 - t_icR_128K_8K_l (Timeout)
	135 - t_icR_128K_8K_L (Failed)
	136 - t_icR_128K_8K_lL (Failed)
	137 - t_icD_128K_8K_l (Failed)
	138 - t_icD_128K_8K_L (Failed)
	139 - t_icD_128K_8K_lL (Failed)
	140 - t_isS_128K_8K_l (Timeout)
	141 - t_isS_128K_8K_L (Failed)
	142 - t_isS_128K_8K_lL (Failed)
	143 - t_isR_128K_8K_l (Failed)
	144 - t_isR_128K_8K_L (Failed)
	145 - t_isR_128K_8K_lL (Failed)
	146 - t_isD_128K_8K_l (Failed)
	147 - t_isD_128K_8K_L (Timeout)
	148 - t_isD_128K_8K_lL (Failed)
	149 - t_icnS_128K_8K_l (Failed)
	150 - t_icnS_128K_8K_L (Failed)
	151 - t_icnS_128K_8K_lL (Timeout)
	152 - t_icnR_128K_8K_l (Timeout)
	153 - t_icnR_128K_8K_L (Timeout)
	154 - t_icnR_128K_8K_lL (Timeout)
	155 - t_icnD_128K_8K_l (Failed)
	156 - t_icnD_128K_8K_L (Failed)
	157 - t_icnD_128K_8K_lL (Failed)
	158 - t_isnS_128K_8K_l (Timeout)
	159 - t_isnS_128K_8K_L (Timeout)
	160 - t_isnS_128K_8K_lL (Timeout)
	161 - t_isnR_128K_8K_l (Timeout)
	162 - t_isnR_128K_8K_L (Timeout)
	163 - t_isnR_128K_8K_lL (Timeout)
	164 - t_isnD_128K_8K_l (Timeout)
	165 - t_isnD_128K_8K_L (Timeout)
	166 - t_isnD_128K_8K_lL (Timeout)
Errors while running CTest
CMakeFiles/check_lab4.dir/build.make:57: recipe for target 'CMakeFiles/check_lab4' failed
make[3]: *** [CMakeFiles/check_lab4] Error 8
CMakeFiles/Makefile2:131: recipe for target 'CMakeFiles/check_lab4.dir/all' failed
make[2]: *** [CMakeFiles/check_lab4.dir/all] Error 2
CMakeFiles/Makefile2:138: recipe for target 'CMakeFiles/check_lab4.dir/rule' failed
make[1]: *** [CMakeFiles/check_lab4.dir/rule] Error 2
Makefile:155: recipe for target 'check_lab4' failed
make: *** [check_lab4] Error 2
